core/orchestrator.go

```diff
@@ -517,49 +514,58 @@ type RemoteTranscoderManager struct {
 func (rtm *RemoteTranscoderManager) Register(transcoder *RemoteTranscoder) {
 	rtm.RTmutex.Lock()
 	defer rtm.RTmutex.Unlock()
+
 	rtm.liveTranscoders[transcoder.stream] = transcoder
 
 	for i := 0; i < transcoder.Capacity(); i++ {
-		rtm.AppendTranscoder(transcoder)
-	}
-}
-
-func (rtm *RemoteTranscoderManager) AppendTranscoder(transcoder *RemoteTranscoder) {
```

Comment:
These changes to the RemoteTranscoderManager can probably be fixed up into https://github.com/livepeer/go-livepeer/pull/833/commits/207e18a84a0fc4603ebab83bcb477a469eddcad8 ?